### PR TITLE
Remove distribution section

### PR DESCRIPTION
**Pull request by adrn**
This is an alternate to #4. I removed the distribution subsection and added a sentence to the end of the release section.

cc @bsipocz @hamogu @astrofrog

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -361,9 +328,9 @@ \section{Astropy Core Package v2.0}
 packages to utilize for a common interface (e.g., \texttt{NDData}).
 In this section, we highlight new features introduced or substantially improved
 since version v0.2 (previously described in \citealt{astropy}).  The \astropypkg
-provides a full log of changes \footnote{https://github.com/astropy/astropy/blob/master/CHANGES.rst}
+provides a full log of changes \footnote{\url{https://github.com/astropy/astropy/blob/master/CHANGES.rst}}
```
Comment (bsipocz) [inline main.tex]: shouldn't this be "stable" rather than master?

File: main.tex [hunk 2]
```diff
@@ -299,44 +299,11 @@ \subsection{Release cycle and Long Term Support}
...
-%
-% > re-built binary conda packages
-% Feel free to remove if this is too detailed. I do think the available of the whole ecosystem across all platforms has helped speed its adoption.
-%
-% ^.
+The released versions of the \astropypkg core package are available from several
+of the \python distributions for scientific computing (e.g.,
+\href{http://anaconda.org}{Anaconda}) and from the \python Package Index
+(PyPI).\footnote{See the installation documentation for more information:
+\url{http://docs.astropy.org/en/stable/install.html}}
```
Comment (crawfordsm) [inline main.tex:306]: Can we add something along the following sentence to reflect Matt's point:

Effort has been made to make \astropypkg available and easily installable across all platforms. This has been done through testing against different platforms as part of our suite of continuous integration tests and inclusion of pre-built versions in different package managers.

File: main.tex (hunk 1 above)
Comment (crawfordsm) [reply main.tex]: switching to stable would be fine with me -- and more appropriate for the v2.0 release

File: main.tex (hunk 2 above)
Comment (adrn) [reply main.tex:306]: OK I added something in this vein - feel free to edit more if you want.

**Comment by adrn** [comment]
Merging this now so it appears in the version sent out by @crawfordsm, but please open an issue or PR if you'd like to edit.

**Pull request merged**